README.md

```diff
@@ -233,6 +233,7 @@ It's a great way to learn.
 * [**C**: _Beej's Guide to Network Programming_](http://beej.us/guide/bgnet/)
 * [**C**: _Let's code a TCP/IP stack_](http://www.saminiir.com/lets-code-tcp-ip-stack-1-ethernet-arp/)
 * [**Ruby**: _How to build a network stack in Ruby_](https://medium.com/geckoboard-under-the-hood/how-to-build-a-network-stack-in-ruby-f73aeb1b661b)
+* [**C / Python**: _Build your own VPN/Virtual Switch_](https://github.com/peiyuanix/build-your-own-zerotier)
```

Comment:
Can we place this above the Ruby item please? (items are sorted by language)

Reply:
OK

Reply:
> Can we place this above the Ruby item please? (items are sorted by language)

I have adjusted the position of this item.
@rohitpaulk